charts/controller/values.yaml

```diff
@@ -28,11 +28,11 @@ controller:
   ##
   resources:
     requests:
-      cpu: 30m
+      cpu: 1
       memory: 32Mi
     limits:
-      cpu: 4000m
-      memory: 1024Mi
+      cpu: 2
+      memory: 4Gi
```

Comment:
memory limit is much bigger than the request, it will cause scheduling on low resources  workers and  evictions when memory consumption will high

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/  Guaranteed QoS is the best, Burstable will also go well but not with a such big difference.